projects/ngrx.io/content/guide/migration/v20.md

```diff
@@ -20,3 +24,250 @@ Version 20 has the minimum version requirements:
 - RxJS version ^6.5.x || ^7.5.0
 
 ## Breaking changes
+
+### Signals
```

Comment:
I summarized breaking changes for both `signalState` and `signalStore` in the body of this commit message: https://github.com/ngrx/platform/commit/521a2a65346234fc7687daa4a8b505d48216b533

We can use its content on the migration page.

Reply:
Sorry, I didn't see that. I updated the with the commit message.